docs/releases/stable/index.md

```diff
@@ -169,6 +169,25 @@ them resolved prior to GA.
 Please note the [Conflict Resolution Section](#conflict-resolution) for making
 any changes as a result of SQA testing.
 
+### Azure Arc Conformance
+
+We certify Charmed Kubernetes with the Azure Arc program by running the following
+job on each new release:
+
+https://jenkins.canonical.com/k8s-ps5/view/Conformance/job/conformance-arc-ck/
+
+This job runs weekly, but publishing the results to Microsoft's bucket is not done
+by default. Ensure we have passing results, and run the above job with the
+"UPLOAD_RESULTS" parameter checked.
+
+Credentials for this job are exported by the `azure-arc.sh` script, defined in the
+"Juju Data - Azure ARC" BitWarden entry, and delivered to all Jenkins workers as part of
+the `juju_creds` [credential](https://jenkins.canonical.com/k8s-ps5/manage/credentials/).
+
+> **Note**: The `AZ_STORAGE_ACCOUNT_SAS` key expires monthly and will need to be
+rotated via [the partner credential portal](https://forms.office.com/pages/responsepage.aspx?id=v4j5cvGGr0GRqy180BHbR9r2AMIPNzpPnFQdZ9IWxshUOFpaWlQ1MkdVRUpBWEtaWU1UUkZJVlA4UCQlQCN0PWcu);
```

Comment:
Should this partner credential portal link be hidden, I don't think this link is protected by SSO?

Reply:
@louiseschmidtgen gone but not dead ;)

that isn't a partner credential in the url there.  that's just the id of the form from office 365 afaik.  the actual ident and secret have to be manually entered into that form by a team member.